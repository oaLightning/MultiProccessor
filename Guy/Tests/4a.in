game_mode 2
start
move <2,E> to <4,E>
move <8,G> to <6,F>
move <4,E> to <5,E>
move <6,F> to <5,D>
move <1,G> to <3,F>
move <7,D> to <6,D>
move <1,F> to <4,C>
move <8,C> to <6,E>
move <1,B> to <3,C>
save 1.xml
quit