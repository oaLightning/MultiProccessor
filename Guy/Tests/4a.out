Specify game setting or type 'start' to begin a game with the current setting:
Game mode is set to 2 players
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m m m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
white player - enter your move:
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
black player - enter your move:
8| R N B Q K B _ R |
7| M M M M M M M M |
6| _ _ _ _ _ N _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
white player - enter your move:
8| R N B Q K B _ R |
7| M M M M M M M M |
6| _ _ _ _ _ N _ _ |
5| _ _ _ _ m _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
black player - enter your move:
8| R N B Q K B _ R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ N m _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
white player - enter your move:
8| R N B Q K B _ R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ N m _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ n _ _ |
2| m m m m _ m m m |
1| r n b q k b _ r |
  -----------------
   A B C D E F G H
black player - enter your move:
8| R N B Q K B _ R |
7| M M M _ M M M M |
6| _ _ _ M _ _ _ _ |
5| _ _ _ N m _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ n _ _ |
2| m m m m _ m m m |
1| r n b q k b _ r |
  -----------------
   A B C D E F G H
white player - enter your move:
8| R N B Q K B _ R |
7| M M M _ M M M M |
6| _ _ _ M _ _ _ _ |
5| _ _ _ N m _ _ _ |
4| _ _ b _ _ _ _ _ |
3| _ _ _ _ _ n _ _ |
2| m m m m _ m m m |
1| r n b q k _ _ r |
  -----------------
   A B C D E F G H
black player - enter your move:
8| R N _ Q K B _ R |
7| M M M _ M M M M |
6| _ _ _ M B _ _ _ |
5| _ _ _ N m _ _ _ |
4| _ _ b _ _ _ _ _ |
3| _ _ _ _ _ n _ _ |
2| m m m m _ m m m |
1| r n b q k _ _ r |
  -----------------
   A B C D E F G H
white player - enter your move:
8| R N _ Q K B _ R |
7| M M M _ M M M M |
6| _ _ _ M B _ _ _ |
5| _ _ _ N m _ _ _ |
4| _ _ b _ _ _ _ _ |
3| _ _ n _ _ n _ _ |
2| m m m m _ m m m |
1| r _ b q k _ _ r |
  -----------------
   A B C D E F G H
black player - enter your move:
Game was successfully saved
Exiting...
